packages/cli/lang/en.lyaml

```diff
@@ -1115,6 +1121,8 @@ en:
           errors:
             buildIdDoesNotExist: "Build {{ buildId }} does not exist for project {{ projectName }}."
             buildAlreadyDeployed: "Build {{ buildId }} is already deployed."
+        deletePrompt:
+          message: Are you sure you want to remove {{ filePath }} from HubSpot?
```

Comment:
Shouldn't this need double quotes? It's working somehow without them on my machine!? Not sure how the parsing logic for this is done but maybe they're not necessary? Haha

Reply:
I have no clue how I missed the double quotes but apparently [YAML doesn't need quotes at all](https://stackoverflow.com/questions/19109912/yaml-do-i-need-quotes-for-strings-in-yaml) lol. Will fix for consistency.

Reply:
TIL!